src/main/resources/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder/tasks-top-extra.jelly

```diff
@@ -24,16 +24,13 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
          xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt" xmlns:this="this">
-    <j:set var="url" value="${h.getNearestAncestorUrl(request,it)}"/>
-    <j:set var="title" value="${%Folder Computation}"/> <!-- fallback, usually invisible -->
+
     <j:set var="computation" value="${it.computation}"/>
-    <j:if test="${computation != null}">
-        <j:set var="title" value="${computation.displayName}"/>
-    </j:if>
-    <l:task icon="images/24x24/notepad.png" href="${url}/computation/" title="${title}">
+
+    <l:task icon="images/24x24/notepad.png" href="${computation.url}" title="${computation.displayName}">
         <j:if test="${h.hasPermission(it,it.BUILD) and it.buildable}">
-            <l:task icon="images/24x24/clock.png" href="${url}/build?delay=0" post="true" title="${%Run Now}"/>
+            <l:task icon="images/24x24/clock.png" href="${computation.url}build?delay=0" post="true" title="${%Run Now}"/>
```

Comment:
@jglick `computation.url` is equivalent to `computation.getUrl()` and this method returns:

```
public @Nonnull String getUrl() {
    return folder.getUrl() + "computation/";
}
```
I don't know that Jelly var.